extensions/ql-vscode/src/databases/local-databases-ui.ts

```diff
@@ -594,6 +614,11 @@ export class DatabaseUI extends DisposableObject {
 
   private async handleChangeLanguageFilter(languageFilter: LanguageFilter) {
     this.treeDataProvider.languageFilter = languageFilter;
+    await this.app.commands.execute(
```

Comment:
This probably conflicts a little with https://github.com/github/vscode-codeql/pull/2873 but I am fine with the PRs going in either order.